docs/effects/api.md

```diff
@@ -93,6 +93,51 @@ export class SomeEffectsClass {
 }
 ```
 
+### Initializing effect
+You can execute some code that will be executed directly after the effect class is loaded.
+```ts
+import { Injectable } from '@angular/core';
+import { Actions, Effect, ofType } from '@ngrx/effects';
+import { defer } from 'rxjs/observable/defer';
+
+@Injectable()
+export class SomeEffectsClass {
+  constructor(private actions$: Actions) { }
+
+  @Effect({ dispatch: false }) init$: Observable<any> = defer(() => {
+    return of(console.log('init$'));
```

Comment:
just use `tap` and `console.log` here. Returning an observable of the console log seems a bit confusing.